modules/nwt-gateway-mode.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+
+:_mod-docs-content-type: PROCEDURE
+[id="nwt-gateway-mode_{context}"]
+= Setting local and shared gateway modes
+
+As a cluster administrator you can configure the gateway mode using the `gatewayConfig` spec in the Cluster Network Operator. The following procedure can be used to set the `routingViaHost` field to `true` for local mode or  `false` for shared mode.
+
+You might follow the optional step 4 to enable IP forwarding alongside local gateway mode if you require node level traffic control. For example, possible use cases for combining local gateway mode with IP forwarding include:
+
+* Configuring all pod egress traffic to be forwarded via the node's IP
+
+* Integrating OVN-Kubernetes CNI with external network address translation (NAT) devices
+
+* Configuring OVN-Kubernetes CNI to use a kernel routing table
+
+
+.Prerequisites
+* You are logged in as a user with admin privileges.
+
+.Procedure
+
+. Back up the existing network configuration by running the following command:
++
+[source,terminal]
+----
+$ oc get network.operator cluster -o yaml > network-config-backup.yaml
+----
+
+. Set `routingViaHost` to `true` for local gateway mode by running the following command:
++
+[source,terminal]
+----
+$ oc patch networks.operator.openshift.io cluster --type=merge -p '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"gatewayConfig":{"routingViaHost": true}}}}}'
+----
+
+. Verify that local gateway mode has been set by running the following command:
++
+[source,terminal]
+----
+$ oc get networks.operator.openshift.io cluster -o yaml | grep -A 5 "gatewayConfig"
+----
++
+.Example output
+[source,terminal]
+----
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+# ...
+gatewayConfig:
+        ipv4: {}
+        ipv6: {}
+        routingViaHost: true <1>
+      genevePort: 6081
+      ipsecConfig:
+# ...
+----
+<1> A value of `true` sets local gateway mode and a value of `false` sets shared gateway mode.
```

Comment:
It may be easier for customers to understand if we bring the "shared gateway" and "logical gateway" to the background and favor talking about  "we are routing through the host" or "we aren't routing through the host". I am not talking about removing the terminology (because users know it already) but easing the explanation that way.

Reply:
done